docs/use/createBackups/EnablePITR.md

```diff
@@ -31,18 +31,95 @@ To enable PITR:
 
 === "When creating a new database"
 
+
     1. On the Everest homepage, click **Create Database** to display the database creation wizard.
     2. Fill in the details of your database on the first steps of the wizard.
     3. Navigate to the **Backups** page, and make sure to enable and configure a backup schedule. 
     4. Click **Next** to go to the **Point-in-time recovery (PITR)** page where you can enable PITR and specify a location for storing the PITR backups and logs. 
     5. Complete the setup wizard to create the new database with PITR enabled. 
 
 === "When editing an existing database"
+
+
     1. In the <i class="uil uil-database"></i> **Databases** view, select the database for which you want to enable PITR.
     2. Click the <i class="uil uil-ellipsis-h"></i> Actions menu next to the database, then click **Edit**.
     3. Navigate through the database configuration wizard to get to the **Backups** page where you can configure a backup schedule that will create an initial full backup required for PITR.
     4. Complete the setup wizard to schedule a full backup and enable PITR for the existing database.
 
+
+## Limitation
+
+When performing point-in-time recovery (PITR) for PostgreSQL, it is important to consider the following limitation:
+
+You may encounter issues with point-in-time recovery (PITR) when attempting to recover the database after the last transaction. PITR can get stuck in the **Restoring** state.
+
+
+**Check the timestamp of the last transaction**
+
+Connect to your database and run the following command. It contains the `gtid`, the `timestamp` and the status of the last transaction.
```

Comment:
> run the following command. It contains the `gtid`, the `timestamp` and the status of the last transaction.

The command itself does not contain those things, the output of the command does.